src/cobra/io/web/bigg_models_repository.py

```diff
@@ -0,0 +1,84 @@
+"""Provide a concrete implementation of the BioModels repository interface."""
+
+
+from io import BytesIO
+from typing import Union
+
+import httpx
+
+from .abstract_model_repository import AbstractModelRepository
+
+
+class BiGGModels(AbstractModelRepository):
```

Comment:
I like this design. Would be great if we could a DB package that adds more like AGORA and CarveME.

Reply:
:+1: thanks. I definitely had in mind that more interfaces can be added.

Reply:
Do you mean a separate Python package entirely or a sub-package within cobrapy?

Reply:
Separate package.